### ISSUE TITLE
Startup Applications has double entry in Mint Menu when more than one user exists

### ISSUE THREAD

**Issue by CottonEaster** [linuxmint/Cinnamon#4843]
Please see screenshot.
![screenshot-screen-2015-11-19-184624](https://cloud.githubusercontent.com/assets/9998846/11280503/322b094e-8eee-11e5-9d46-a40c34e889d3.png)

The two entries do _not_ seem to correspond to the two users' separate startup entries.

**Comment by JosephMcc** [comment]
I get this on my ubuntu install where I have two different desktops installed. One is cinnamon-settings startup and the other in gnome-session-properties.

**Comment by ManIVIctorious** [comment]
If this is just an other entry with the same name, but launching a different program I wouldn't call this an issue, since the menu is doing exactly as expected.
@ACinnamonUserInUK can you confirm if both entries start the same application?

**Comment by ACinnamonUserInUK** [comment]
Both entries _do_ run the same command, viz. `cinnamon-settings startup`. However, and as the screenshot shows, the icons are different. (I think at one point I did manage somehow to run the 'gnome' version of the, startup application; but that is not what is happening here and now.)

Current Cinnamon version (which may be different from the version I had when I created the bug; certainly, I've changed themes): 3.0.7.

**Comment by Odyseus** [comment]
Hello, g@ys.

@ManIVIctorious, @ACinnamonUserInUK and @JosephMcc : Tested this on Ubuntu Gnome with Cinnamon installed from this repository. The Cinnamon menu lists both **Startup Applications** entries, but they each launch the correct command, no matter which user is logged in (3 different users). In a system with just Cinnamon installed, the menu lists just one **Startup Applications** entry.

**Comment by ACinnamonUserInUK** [comment]
Thanks 'Odyseus'. But I have only one desktop environment (DE) installed, namely, Cinnamon. However, I did, in the past, play around - on the same system - with some other DEs. So perhaps remnants are responsible. Even if so, it might be good if Cinnamon could handle them sensibly.

**Comment by Odyseus** [comment]
@ACinnamonUserInUK: The Cinnamon menu lists desktop files found on your system and already handles them sensibly. It will not list desktop files with invalid commands, for example. If you see two **Startup Applications** entries, it's because you have two desktop files with the exact same **Name=** defined inside them and both files have valid commands defined in **Exec=**.

If what you want is to get rid of the extra **Startup Applications** entry (the one with the blue icon), just delete it's desktop file from **/usr/share/applications**.

**Comment by leigh123linux** [comment]
@Odyseus 

A better solution would be to use the menu editor to uncheck the offending menu item which would hide it.
You method  would break every time the app was updated as the missing desktop file would be replaced.

**Comment by ACinnamonUserInUK** [comment]
Odyseus, leigh123linux

Ultimately the method I used to remove the duplicate menu options was the unchecking option. Initially I went for the deletion option and while I was at it I removed various files on my system that seemed left over from an installation of KDE. Result: my icons got changed to stock Gnome (I think) and would not change back. To fix that, the only think I found that worked (short of reimaging /root and or /home - I didn't resort to that) was to restore the system using the `SystemBack` program.

I did search the system (with `grep` and with `AngrySearch`) for a .desktop file that was causing a second appearance of the 'startup applications' item but I failed to find one.
